lib/rouge/lexers/php.rb

```diff
@@ -65,14 +68,17 @@ def builtins
       def self.keywords
         @keywords ||= Set.new %w(
           and E_PARSE old_function E_ERROR or as E_WARNING parent eval
-          PHP_OS break exit case extends PHP_VERSION cfunction FALSE
+          PHP_OS break exit case extends PHP_VERSION cfunction false
           print for require continue foreach require_once declare return
-          default static do switch die stdClass echo else TRUE elseif
+          default static do switch die stdclass echo else true elseif
```

Comment:
If you took `stdclass` out of here, where would you put it?

Reply:
I see two approaches:

* a rule to specifically handle `stdclass` as a builtin class
* [recognize all predefined classes](https://www.php.net/manual/en/reserved.classes.php)

Reply:
I'm a little unclear on how `stdclass` works (and a very quick search of the PHP site wasn't that helpful). Could you explain it? I'm not sure what the purpose would be of having a rule to specifically handle it.

Reply:
stdClass is more used for PHP's internal stuffs. For developers, it's like an empty shell to which you then dynamically add properties:

```php
$thing = new stdClass();
$thing->foo = 'Hello';
$thing->bar = 'world';
var_dump($thing);
```

```
object(stdClass)#1 (2) {
  ["foo"]=>
  string(5) "Hello"
  ["bar"]=>
  string(5) "world"
}
```

Reply:
Got it.

Reply:
Sorry—I should have made sure I remembered what we were talking about. I am in favour of removing `stdClass` from here and making it a separate rule as you described.

Reply:
Sure, it also makes more sense to me.